crates/notion-core/src/inventory/mod.rs

```diff
@@ -277,6 +287,29 @@ impl FetchResolve<NodeDistro> for NodeCollection {
         }
     }
 
+    fn resolve_lts(
+        &self,
+        _name: String,
+        hooks: Option<&ToolHooks<NodeDistro>>,
+    ) -> Fallible<Version> {
+        let url = match hooks {
+            Some(&ToolHooks {
+                index: Some(ref hook),
+                ..
+            }) => hook.resolve("index.json")?,
+            _ => public_node_version_index(),
+        };
+        let version_opt = match_node_version(&url, |&NodeEntry { lts: ref v, .. }| v.clone())?;
```

Comment:
`lts` is a Boolean value, so I don't think we need to use the `ref v` and `v.clone()`, instead I think this should be doable as just:
`|&NodeEntry { lts: val, .. }| v`